tool-base/src/main/kotlin/io/spine/tools/StandardTypesExts.kt

```diff
@@ -42,5 +43,14 @@ public fun Supplier<String>.toAbsoluteFile(): File = Files2.toAbsolute(get())
 /** Adds relative name to this directory. */
 public fun File.resolve(dir: DirectoryName): File = resolve(dir.value())
 
+/** Adds relative directory to this directory. */
+public operator fun File.div(dir: DirectoryName): File = resolve(dir)
```

Comment:
Instead of `div`, I would go with `sub`, or `slash` (while longer, this is how we would call `/` symbol).

`div` smells HTML-y.